ui/src/TestSuite/TestSuiteList.tsx

```diff
@@ -79,15 +79,17 @@ const TestSuiteList = ({ publicId, testSuites }: TestSuiteListProps) => {
           },
           { title: "Passed", field: "passed", cellStyle, headerStyle },
           { title: "Failed", field: "failed", cellStyle, headerStyle },
-          { title: "Duration", field: "duration", cellStyle, headerStyle },
+          { title: "Duration", field: "duration", render: (rowData) => (
+              <>{rowData.duration}s</>
+            ), cellStyle, headerStyle },
         ]}
         data={testSuites.map((testSuite) => ({
           fileName: testSuite.fileName,
           name: fullTestSuiteName(testSuite),
           group: testSuite.groupName || "",
           passed: testSuite.passingCount,
           failed: testSuite.failureCount,
-          duration: `${testSuite.duration}s`,
+          duration: testSuite.duration,
```

Comment:
want to keep this as numeric so the native sorting of the table library we use will sort it. a `render` function was added above to format the duration.